gradle.properties

```diff
@@ -5,10 +5,13 @@ bintrayRepoType=defineMe
 
 eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-release/,https://dl.bintray.com/eventuateio-oss/eventuate-maven-rc/,file:///Users/cer/.m2/testdeploy
 
-springBootVersion=2.0.3.RELEASE
-eventuateClientVersion=0.20.1.RELEASE
-eventuateTramVersion=0.20.0.RELEASE
-eventuateLocalVersion=0.30.0.RELEASE
+springBootVersion=2.1.1.RELEASE
+eventuateClientVersion=0.20.0-SNAPSHOT
+eventuateTramVersion=0.20.0.REDIS-SNAPSHOT
```

Comment:
I don't think REDIS-SNAPSHOT is the current one?

Reply:
It seems now it is 0.21.0-SNAPSHOT. I will merge latest changes to tram.